client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { translate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Card, Button } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelCta from 'components/action-panel/cta';
+import CardHeading from 'components/card-heading';
+import { getTaskList } from 'lib/checklist';
+import Gridicon from 'components/gridicon';
+import getSiteChecklist from 'state/selectors/get-site-checklist';
+import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
+import { getSiteOption } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import NavItem from './nav-item';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const getTaskCopy = ( task, taskOptions ) => {
+	switch ( task ) {
+		case 'domain_verified':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 2, args: [ 2 ] } ),
+				title: translate( 'Verify the email address for your domains' ),
+				description: translate(
+					'We need to check your contact information to make sure you can be reached. Please verify your details using the email we sent you, or your domain will stop working.'
+				),
+			};
+		case 'email_verified':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 1, args: [ 1 ] } ),
+				title: translate( 'Confirm your email address' ),
+				description: translate(
+					'Please click the link in the email we sent to %(email)s.' +
+						'Typo in your email address? {{changeButton}}Change it here{{/changeButton}}.',
+					{
+						args: {
+							email: taskOptions?.userEmail,
+						},
+						components: {
+							br: <br />,
+							changeButton: <a href="/me/account" />,
+						},
+					}
+				),
+				//TODO: looks like there's some more complicated text states here
+				actionText: translate( 'Verify email' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Verify email first' )` *ES Score: 7*